src/output/plugins/PulseOutputPlugin.cxx

```diff
@@ -23,6 +23,10 @@
 
 #include <stdlib.h>
 
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
According to MSDN, the function is provided by `processenv.h`; we should always use the narrowest possible header.

Reply:
OTOH, upnp support is broken because of that.

Reply:
Fixed.